tests/my-game/example/ability.ts

```diff
@@ -0,0 +1,73 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
This seems to add files we already have, but under a slightly different dir name?

Reply:
That's how the new TS generator works. Test files were not added along with that change, so I added them now. Seems like we need to clean up the test directory at some point.

Reply:
Why though? The namespace in the schema is `MyGame`, and all other languages generate files in the `MyGame` dir. Don't think it would be helpful to have a parallel `my-game` dir just for TS?

Also, may need to delete `monster_test_generated.js/.ts`

Reply:
TS now generates files just like java, so the directory matches the namespace.

Reply:
The Java files are in `MySpace`, why does TS need to change the namespace?

Reply:
https://github.com/google/flatbuffers/blob/master/tests/MyGame/Example/Ability.java is in `MyGame.Example`, just like this file is.

Reply:
I'll split test cleanup from he actual fix.

Reply:
@aardappel it's just the chosen naming convention for module filenames which seem to be the most widespread for ES modules.

Reply:
Ok, but we so far have all PLs supported following the original namespace spelling, not sure why TS should be special here. Or maybe it should be an option?

Reply:
I think it's more natural to follow target language conventions.